content/neuvector-prime/azure/contents.lr

```diff
@@ -169,6 +169,19 @@ Once logged in, you can begin to [navigate and configure NeuVector]. Refer (http
 
    !["Keyvaluepair config"](./keyvaluepairconfig.png)
 
+
+## Install Scanner Updater
+
+The NeuVector scanner image is updated daily with a new CVE database on the NeuVector docker hub registry.It is recommended to install scanner updater AFTER successfull initial NeuVector deployment.
+
+Execute the following command in a Cloud Shell
+
+```
+helm repo add
+```
+
+
+
 ## Auto upgrade minor version
```

Comment:
this whole section will no longer apply if we're installing the scanner independent of the CNAB... I'm not 100% sure thats been decided

Reply:
done